datahub-dao/src/main/java/com/linkedin/datahub/dao/DaoFactory.java

```diff
@@ -101,4 +103,11 @@ public static LineageDao getLineageDao() {
     }
     return lineageDao;
   }
+
+  public static DataPlatformsDao getDataPlatformsDao() {
+    if (dataPlatformsDao == null) {
+      dataPlatformsDao = new DataPlatformsDao(getGmsDao().get_dataPlatforms());
```

Comment:
Why does this have an _ in the middle?

Reply:
It's an automatically generated getter by lombok.